dune-workspace.dev

```diff
@@ -1,4 +1,4 @@
 (lang dune 1.0)
 
 ;; This file is used by `make all-supported-ocaml-versions`
-(context (opam (switch 4.08.0)))
+(context (opam (switch 4.08.1)))
```

Comment:
I personally don't use this workspace file. So feel free to delete it unless you're using it.

Reply:
Ack, I removed it.